### PR TITLE
website assignment project

### PR DESCRIPTION
**Pull request by ashutoshkumar55**

### CODE REVIEW

**Review by Vedant-29 — CHANGES_REQUESTED** [review]
Ashutosh it was mentioned in the rules that u had to make a website with at least 6-7 web pages. but our website only has one HTML and one CSS page. 

pls add more web pages and also add hyperlinks that can link one web page to another.

File: ashutosh.txt/css/style.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ 	background-repeat: no-repeat;
+ 	background-position: center;
+ 	height: 100vh;
+ 	background-size: cover;
+ }
+ .clearfix:after{
+ 	content: ".";
+ 	visibility: hidden; 
+ 	display: block;
+ 	height:0px;
```
Comment (SnehaMakade) [inline ashutosh.txt/css/style.css:23]: height: after colon there should be space

**Comment by ashutoshkumar55** [comment]
okk mam

Thanks

**Review by Vedant-29 — CHANGES_REQUESTED** [review]
Ashutosh redo the website. 
the home page image is not visible.
offers and gift cards have no link.
copying from others is not acceptable so pls make a new website.
no header and footer on the front page.
the logo not visible.
advance ticket booking Is overlapping.
not links in the left bar menu of the booking page.

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
make correct spelling for commit

**Comment by ashutoshkumar55** [comment]
Done mam

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
Do proper indentation
2. in add server entry point in package.json
3.do nodemon install 
4. At localhost:3000 show added employee list

**Comment by ashutoshkumar55** [comment]
Okk mam

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
submit button and viewall button is not working so check your routes.

**Comment by anitapise** [comment]
ashutosh left

**Pull request closed without merging**